### DISCUSSION TITLE
Add The Mirror

### DISCUSSION THREAD

**Discussion by kparal** [opengaming/osgameclones#2656] [General]
The Mirror is a clone of Roblox and Unreal Editor for Fortnite (UEFN). Its source code is officially published.

Homepage: https://www.themirror.space
Source: https://github.com/the-mirror-gdp/the-mirror

**Comment by cxong** [comment]
I'm unsure about adding this to OSGC; roblox and UEFN are not really games, but games creation platforms. They are similar to, but not quite like, game engines or even game frameworks. The mirror looks like an interesting project, but given the above I'm not sure OSGC is the right place for it.

**Comment by kparal** [comment]
That's fair. I don't have an opinion on this.
